wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/json/jackson/AliasesDeserializer.java

```diff
@@ -80,8 +80,6 @@
 				}
 			}
 
-		} catch (JsonProcessingException e) {
-			e.printStackTrace();
 		} catch (Exception e) {
```

Comment:
Right, sure, I agree it is bad practice to catch Exception. Those changes I'm making here are done with very minimal understanding of the context, and are generally strictly refactoring (ie no behavior changes).